.github/workflows/linter.yml

```diff
@@ -1,14 +1,11 @@
+---
 name: super-linter
```

Comment:
More a general topic: In the cicd action we do most of the linter/type checking stuff. Do we still need super-linter in the OTC repositories?